templates/commands/render_action_print.go

```diff
@@ -45,11 +42,11 @@ func actionPrint(ctx context.Context, p *model.Print, sp *stepParams) error {
 	return nil
 }
 
-func flagsForTemplate(r *RenderFlags) map[string]any {
+func flagsForTemplate(r *RenderFlags) map[string]string {
 	// We only expose certain fields the print action; these are the ones that
 	// we have beneficial use cases for and that don't encourage bad API use.
-	return map[string]any{
-		"dest":   r.Dest,
-		"source": r.Source,
+	return map[string]string{
+		"_flag_dest":   r.Dest,
+		"_flag_source": r.Source,
```

Comment:
The leading underscore is to indicate that it's a reserved namespace. The underscore in the middle it because I need the go-template engine to see it as a single symbol, so we can't use `.` or `-`.